README.md

```diff
@@ -1,43 +1,42 @@
-# repo-report <sup>[![Version Badge][npm-version-svg]][package-url]</sup>
+# 🧾 Repo-Report
 
-[![github actions][actions-image]][actions-url]
-[![coverage][codecov-image]][codecov-url]
-[![dependency status][deps-svg]][deps-url]
-[![dev dependency status][dev-deps-svg]][dev-deps-url]
-[![License][license-image]][license-url]
-[![Downloads][downloads-image]][downloads-url]
```

Comment:
with you deleting this, the documentation is losing all of the following information:

<img width="834" height="311" alt="image" src="https://github.com/user-attachments/assets/a060507e-8801-4ff2-bbb5-863577e066e5" />

can we only delete the ones that are broken and keep the ones that are working?